From d00cd70e561e65fe9b0968a378ef247b85026542 Mon Sep 17 00:00:00 2001
From: Sudip Mukherjee <sudipm.mukherjee@gmail.com>
Date: Fri, 6 Nov 2020 15:37:44 +0000
Subject: [PATCH 11/12] driver core: export device_is_bound() to fix build
 failure

When CONFIG_MXC_CLK_SCU is configured as 'm' the build fails as it
is unable to find device_is_bound(). The error being:
ERROR: modpost: "device_is_bound" [drivers/clk/imx/clk-imx-scu.ko]
	undefined!

Export the symbol so that the module finds it.

Signed-off-by: Sudip Mukherjee <sudipm.mukherjee@gmail.com>
Link: https://lore.kernel.org/r/20201106153744.22661-1-sudipm.mukherjee@gmail.com
Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 drivers/base/dd.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/base/dd.c b/drivers/base/dd.c
index 148e81969e04..a796a57e5efb 100644
--- a/drivers/base/dd.c
+++ b/drivers/base/dd.c
@@ -353,6 +353,7 @@ bool device_is_bound(struct device *dev)
 {
 	return dev->p && klist_node_attached(&dev->p->knode_driver);
 }
+EXPORT_SYMBOL(device_is_bound);
 
 static void driver_bound(struct device *dev)
 {
-- 
2.29.2

